src/nmo.js

```diff
@@ -49,6 +49,8 @@ nmo.load = function load (opts) {
       .load(opts)
       .then((config) => {
         nmo.config = config;
+        nmo.config.set('name', 'nmo');
+        nmo.config.set('help-directory', `${__dirname}/..`);
```

Comment:
hmmm...

what do you think of adding them as defaults to the nmo conf file that is placed into the home dir?

this smells dangerously for race conditions